src/components/Post/Createpost.js

```diff
@@ -3,10 +3,15 @@ import React from 'react';
 import './PostComponent.css';
 import Button from '../Shared/Button/Button';
 import './CreatePost.css';
```

Comment:
minor detail, but this file should be CreatePost.js to match our current file naming convention